extension/scripts/features/last-action/ttFactionLastAction.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+(async () => {
+	const feature = featureManager.registerFeature(
+		"Last Action - Faction",
+		"last action",
+		() => settings.scripts.lastAction.factionMember,
+		addListener,
+		addLastAction,
+		removeLastAction,
+		{
+			storage: ["settings.scripts.lastAction.factionMember"],
+		},
+		null
+	);
+
+	function addListener() {
+		if (isOwnFaction()) {
+			CUSTOM_LISTENERS[EVENT_CHANNELS.FACTION_INFO].push(async () => {
+				if (!feature.enabled()) return;
+
+				await addLastAction(true);
+			});
+		}
+	}
+
+	async function addLastAction(force) {
+		if (isOwnFaction() && !force) return;
+		if (document.find(".tt-last-action")) return;
+		await requireElement(".members-list .table-body > li");
+		const lastActionList = (await fetchData("torn", { section: "faction", ...isOwnFaction() ? {} : { id: parseInt(getSearchParameters().get("ID")) } })).members;
```

Comment:
Did you test this? I'm pretty sure that this will try to spread `true` or `false`, which throws an error.

Reply:
Yes.